ocaml/idl/datamodel.ml

```diff
@@ -641,6 +641,8 @@ let _ =
     ~doc:"VGPU type is not one of the PGPU's supported types." ();
   error Api_errors.vgpu_type_not_compatible_with_running_type ["pgpu"; "type"; "running_type"]
     ~doc:"VGPU type is not compatible with one or more of the VGPU types currently running on this PGPU" ();
+  error Api_errors.vgpu_not_compatible_with_destination_pgpu ["reason"]
```

Comment:
I think you are right, even if (as it is currently) I check only the `scheduled_to_be_resident_on` pgpu for the pool_migration assert